src/map/Map.cpp

```diff
@@ -161,12 +162,31 @@ void Map::genWalls(const vector<MapPoint>& wallStart) {
  * Function Interface: void mapLoadToGame()
  * Description:
  * Creates the wall structures using the GameManager createWall function.
+ * Creates and loads shop and dropzone positions.
  */
 void Map::mapLoadToGame() {
+    // Random shop position variable.
+    int pos;
+    // Random number generator.
+    mt19937 ran;
+    // Shop position being loaded.
+    MapPoint shopPosition;
+
     for (const auto& w : walls) {
-        logv("Creating wall");
         GameManager::instance()->createWall(w.x, w.y, w.width, w.height);
     }
+
+    // Random number generation.
+    ran.seed(1);
```

Comment:
I removed the seed, but it still gives me the same number each time it runs... not too sure why, but we can fix it later.